desktop-widgets/diveplanner.cpp

```diff
@@ -117,6 +118,9 @@ DivePlannerWidget::DivePlannerWidget(QWidget *parent, Qt::WindowFlags f) : QWidg
 	ui.cylinderTableWidget->setTitle(tr("Available gases"));
 	ui.cylinderTableWidget->setBtnToolTip(tr("Add cylinder"));
 	ui.cylinderTableWidget->setModel(CylindersModel::instance());
+	ui.waterType->setItemData(0, (double)FRESHWATER_SALINITY/10000.0);
```

Comment:
1) No point in casting to `double` (the result of division by 10000.0 is `double` by definition).
2) It would be more consistent to do all internal arithmetics in `int` and convert only on input/output.

Reply:
Ok.